lib/airbrake/rake/tasks.rb

```diff
@@ -100,7 +100,10 @@
 
     exit!(1) if [id, key].any?(&:nil?)
 
+    env = heroku_env['RAILS_ENV'] || ENV['RAILS_ENV']
```

Comment:
Simply `heroku` won't cut it because we need to be able to resolve errors automatically, so the value should match with what users have. How about:

```ruby
env = heroku_env['RAILS_ENV'] || 'production'
```